app/src/pages/authoring/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button, Group, Radio, Title } from '@mantine/core';
+import { v4 as uuidv4 } from 'uuid';
+import { useState } from 'react';
+import { trpc } from '../../util/trpc';
+import { MeasureSkeleton, LibrarySkeleton } from '@/util/authoringFixtures';
+import { useRouter } from 'next/router';
+import { notifications } from '@mantine/notifications';
+import { AlertCircle, CircleCheck } from 'tabler-icons-react';
+import { ArtifactResourceType } from '@/util/types/fhir';
+
+export default function AuthoringPage() {
+  const [resourceType, setResourceType] = useState<ArtifactResourceType>('Measure');
+  const draftMutation = trpc.createDraft.useMutation({
+    onSuccess: data => {
+      notifications.show({
+        title: `${resourceType} Created!`,
+        message: `${resourceType} successfully created`,
+        icon: <CircleCheck />,
+        color: 'green'
+      });
+      router.push(`authoring/${resourceType}/${data.draftId}`);
+    },
+    onError: e => {
+      notifications.show({
+        title: `${resourceType} Creation Failed!`,
+        message: `Attempt to create ${resourceType} failed with message: ${e.message}`,
+        icon: <AlertCircle />,
+        color: 'red'
+      });
+    }
+  });
+
+  const router = useRouter();
+
+  const createResource = () => {
+    const newResource = resourceType === 'Measure' ? { ...MeasureSkeleton } : { ...LibrarySkeleton };
+    newResource.id = uuidv4();
+    draftMutation.mutate({ resourceType, draft: newResource });
+  };
+
+  return (
+    <Group
+      spacing="lg"
+      position="left"
+      style={{
+        flexDirection: 'column',
+        width: '100%',
+        alignItems: 'flex-start',
+        justifyContent: 'flex-start'
+      }}
+    >
+      <Radio.Group
+        label="Resource Type"
+        value={resourceType}
+        onChange={val => setResourceType(val as ArtifactResourceType)}
+      >
+        <Group>
+          <Radio label="Measure" value="Measure" />
+          <Radio label="Library" value="Library" />
+        </Group>
+      </Radio.Group>
```

Comment:
Thoughts on using a [SegmentedControl](https://mantine.dev/core/segmented-control/) instead of Radio? Might look a little sleeker.